commit 650a94ff6f207b0bb928f631b80a3787e446718c
Author: Maxime Ripard <maxime.ripard@free-electrons.com>
Date:   Mon Jan 23 11:41:48 2017 +0100

    rtc: sun6i: Switch to the external oscillator
    
    commit fb61bb82cb46a932ef2fc62e1c731c8e7e6640d5 upstream.
    
    The RTC is clocked from either an internal, imprecise, oscillator or an
    external one, which is usually much more accurate.
    
    The difference perceived between the time elapsed and the time reported by
    the RTC is in a 10% scale, which prevents the RTC from being useful at all.
    
    Fortunately, the external oscillator is reported to be mandatory in the
    Allwinner datasheet, so we can just switch to it.
    
    Fixes: 9765d2d94309 ("rtc: sun6i: Add sun6i RTC driver")
    Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1344_650a94ff6f207b0bb928f631b80a3787e446718c.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1344_650a94ff6f207b0bb928f631b80a3787e446718c.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
